### PR TITLE
Update meta-browser to the latest snapshot

### PR DESCRIPTION
**Pull request by msisov**
Added: touch support, start chromium with URL specified in command line,
stability fixes and etc.

Recipe changes: remove use_gold, start using use_jumbo_build, which
reduces compilation time

What is more, added recipe for nss_3.26, which is the least required
version by chromium.

### CODE REVIEW

File: recipes-browser/chromium/chromium-browser-gn.inc [hunk 1]
```diff
@@ -127,10 +125,10 @@ EXTRA_OEGN = " \
         fatal_linker_warnings=false  \
         v8_use_external_startup_data=false \
         linux_use_bundled_binutils=false \
-        use_gold=true \
```
Comment (tonikitoo) [inline recipes-browser/chromium/chromium-browser-gn.inc:130]: maybe adding use_gold=false to state clearly we are disabling it?

File: recipes-browser/chromium/chromium/chromium/0001-Fix-set-is-already-declared-in-this-scope.patch [hunk 2]
```diff
@@ -0,0 +1,33 @@
+From a4878a2c223b4f49d30d7287b745f9546e2a5697 Mon Sep 17 00:00:00 2001
+From: Maksim Sisov <msisov@igalia.com>
+Date: Fri, 11 Aug 2017 11:50:21 +0300
+Subject: [PATCH] Fix "set is already declared in this scope"
+
+../../chrome/browser/devtools/devtools_file_system_indexer.cc:37:12: error: 'set' is already declared in this scope
```
Comment (tonikitoo) [inline recipes-browser/chromium/chromium/chromium/0001-Fix-set-is-already-declared-in-this-scope.patch:6]: lets upstream this next week.

File: recipes-browser/chromium/chromium/chromium/0002-Fix-void-BoxLayoutExample-RefreshLayoutPanel-is-priv.patch [hunk 3]
```diff
@@ -0,0 +1,49 @@
+From 66d6b922b1e147a4f9a862229146f51c69fe7d54 Mon Sep 17 00:00:00 2001
+From: Maksim Sisov <msisov@igalia.com>
+Date: Fri, 11 Aug 2017 13:00:48 +0300
+Subject: [PATCH] Fix 'void BoxLayoutExample::RefreshLayoutPanel()' is private
```
Comment (tonikitoo) [inline recipes-browser/chromium/chromium/chromium/0002-Fix-void-BoxLayoutExample-RefreshLayoutPanel-is-priv.patch:4]: lets upstream this next week.

**Review by tonikitoo — APPROVED** [review]

**Comment by tonikitoo** [comment]
the nss patches look a bit unfortunate, but needed. any chance to get rid of them soon?

**Comment by msisov** [comment]
tonikitoo, I will make a request to AGL and ask that.

File: recipes-browser/chromium/chromium-browser-gn.inc (hunk 1 above)
Comment (msisov) [reply recipes-browser/chromium/chromium-browser-gn.inc:130]: done

**Pull request merged**